Mathlib/Data/Finset/Prod.lean

```diff
@@ -93,6 +93,15 @@ theorem product_subset_product_left (hs : s ⊆ s') : s ×ˢ t ⊆ s' ×ˢ t :=
 theorem product_subset_product_right (ht : t ⊆ t') : s ×ˢ t ⊆ s ×ˢ t' :=
   product_subset_product (Subset.refl _) ht
 
+theorem prodMap_image_product {δ : Type*} [DecidableEq β] [DecidableEq δ]
```

Comment:
Do we usually treat `image` as an infix operator in names or should this start with `image_`?

Reply:
I copied from `piMap_image_pi`. I don't care too much about the names, so feel free to suggest other reasonable names, and I'll accept.